### ISSUE TITLE
[Airsonic] External DBs (7)

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#1756]
## Problem description

This issue was ported from the Airsonic archive. The top node for these issues is #1725.

It does not necessarily aim to solve all problems. However, some points of view will be fed back as Jpsonic is improved in the future.

It also includes issues that have already been resolved by Jpsonic at this stage. These will be progressively organized by topic.

## System information

 * **Jpsonic version**: since v111.5.4

## New task is issued

Prerequisites: #1809

___

 - airsonic/airsonic#1236
   - -> #1810
 - airsonic/airsonic#1340
   - airsonic/airsonic#1299
   - -> #1811
 - airsonic/airsonic#1452
 - airsonic/airsonic#1563
   - -> #1812
 - airsonic/airsonic#1704
 - airsonic/airsonic#1641
   - -> #1813

## Closed

Tracking of the following agenda is closed.

 - airsonic/airsonic#1710
   - Resolved at Airsonic.
 - airsonic/airsonic#1715
   - It is not reproduced in Jpsonic.
